app/api/deputies_api.rb

```diff
@@ -30,6 +30,23 @@ class DeputiesApi < Grape::API
         deputy_activity = DeputyLegislature.where(deputy_id: params[:id], legislature_id: params[:legislature_id]).first
         present deputy_activity, with: Entities::Activity
       end
+
+      desc 'Get deputy' do
+        tags %w[deputies]
+        http_codes [
+          { code: 200, model: Entities::FullDeputy, message: 'Deputy info' },
+          { code: 404, message: 'Deputy not found!' }
+        ]
+      end
+
+      params do
+        requires :legislature_id, type: Integer
+      end
+      get do
+        deputy_legislature = DeputyLegislature.where(deputy_id: params[:id],
+                                                     legislature_id: params[:legislature_id]).first
+        present deputy_legislature.deputy, with: Entities::FullDeputy
```

Comment:
Hm, we might have an issue here if the corresponding `deputy_legislature` is not found. So we can add an early return, wdyt?